src/DynamoCoreWpf/UI/GuidedTour/GuidesValidationMethods.cs

```diff
@@ -569,7 +569,7 @@ internal static void ExecuteViewDetailsSideBar(Step stepInfo, StepUIAutomation u
                 if (packageManagerSearchElementViewModel == null) return;
 
                 if (packageDetailsWindow == null)
-                    packageManagerViewModel.ViewPackageDetailsCommand.Execute(packageManagerSearchElementViewModel.Model);
+                    packageManagerViewModel.ViewPackageDetailsCommand.Execute(packageManagerSearchElementViewModel.SearchElementModel);
```

Comment:
Why the model change here?

Reply:
The `PackageManagerSearchElementViewModel` VM is extending the `BrowserItemViewModel` which also has a `Model` property. So the `PackageManagerSearchElementViewModel` was hiding that property, which would generate an `Ambiguous match error` later down the line. It wouldn't crash and it would work fine, but the error would be there. Renaming that property to a unique name solved the issue.

Here we are just refactoring that rename.